src/index.html

```diff
@@ -1,14 +1,26 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="box-1 boxes">1</div>
+
+      <div class="box-2 boxes">2</div>
+
+      <div class="box-3 boxes">3</div>
+
+      <div class="box-4 boxes">4</div>
+
+      <div class="box-5 boxes">5</div>
+
+      <div class="box-6 boxes">6</div>
```

Comment:
No real need to add empty lines if elements are not multiline.
```suggestion
      <div class="box-1 boxes">1</div>
      <div class="box-2 boxes">2</div>
      <div class="box-3 boxes">3</div>
      <div class="box-4 boxes">4</div>
      <div class="box-5 boxes">5</div>
      <div class="box-6 boxes">6</div>
```